WordPress/Classes/ViewRelated/Reader/ReaderPostDetailViewController.h

```diff
@@ -1,9 +1,11 @@
 #import <UIKit/UIKit.h>
+#import "ReaderPostsViewController.h"
 
 @class ReaderPost;
 
 @interface ReaderPostDetailViewController : UIViewController
 
+@property (nonatomic, assign) ReaderViewStyle readerViewStyle;
```

Comment:
Hmm.  Honestly I'd prefer to remove the `readwrite` declaration on `shouldHideComments`. Feels weird to declare the defaults unless there is a reason beyond aesthetics.